packages/mobile/src/language/Language.tsx

```diff
@@ -54,6 +56,12 @@ export default function Language({ route }: Props) {
   )
 }
 
+Language.navigationOptions = ({ navigation }: ScreenProps) => {
+  return navigation.canGoBack() ? headerWithBackButton : emptyHeader
```

Comment:
This is pretty cool, I was not aware of `canGoBack` method. I solved this issue differently here https://github.com/celo-org/celo-monorepo/pull/4161/files#diff-f418c92450b21cbd54162b941cf091f0R82. But it feels, that your way is slightly better 🙂 So, if my PR will be merged earlier - feel free to resolve conflicts in favour of your approach.

Reply:
Ahah we're all fixing it. I've fixed it too with this approach but applying it everywhere:
https://github.com/celo-org/celo-monorepo/pull/4213/files#diff-5992d9baf806b71ab634873ff02d6b60

Reply:
Jean for the win lol